marimo/_cli/cli.py

```diff
@@ -377,6 +389,7 @@ def edit(
     allow_origins: Optional[tuple[str, ...]],
     skip_update_check: bool,
     sandbox: Optional[bool],
```

Comment:
could make `sandbox` hidden

Reply:
`edit` command is shared for multi-notebook servers and for named edit sessions (`marimo edit foo.py`), so we can't hide `sandbox`.